README.md

```diff
@@ -80,7 +80,7 @@ This repository also includes an
 [introduction to Jsonnet](docs/jsonnetIntro.md).
 
```

Comment:
Can you please remove:  "This repository also includes an [introduction to Jsonnet](docs/jsonnetIntro.md)."